cogs/Util.py

```diff
@@ -2,20 +2,25 @@
 
 import data
 
+
 class Util(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
         self.data = data.Data()
 
-    #Dms are assumed to be elevenlabs keys
+    # Dms are assumed to be elevenlabs keys
     @commands.Cog.listener("on_message")
     async def register_dm(self, message):
         if message.author == self.bot.user:
             return
         if not message.guild:
             self.data.save_eleven_labs_key(message.author.id, message.content)
-            await message.channel.send("Hello " + message.author.name + ", your elevenlabs id has been added")
+            await message.channel.send(
+                "Hello "
+                + message.author.name
+                + ", your elevenlabs id has been added"
+            )
```

Comment:
```suggestion
            await message.channel.send(
                f"Hello {message.author.name}, your ElevenLabs ID has been added."
            )
```
unsure if f-string will work here but I believe so